spec/lib/rabbit_feed/producer_spec.rb

```diff
@@ -68,6 +68,20 @@ module RabbitFeed
           subject
         end
       end
+
+      context 'when RabbitFeed.route_prefix_extension is set' do
```

Comment:
`RabbitFeed.route_prefix_extension` no longer exists - could change to `when a route_prefix_extension is set`